keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -228,6 +233,216 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def text_to_image_prompt_to_prompt(
+        self,
+        prompt: str,
+        prompt_edit: str,
+        method: str,
+        self_attn_steps: Union[float, Tuple[float, float]],
+        cross_attn_steps: Union[float, Tuple[float, float]],
+        attn_edit_weights: np.ndarray = np.array([]),
```

Comment:
this is actually a bug.  If you assign a default value of a function to a single instance and modify that instance it gets modified across calls.

def my_test(x=[]):
    x.append(1)
    print(x)

my_test()
# > [1]
my_test()
# > [1, 1]

Replace with an empty tuple if possible.

Reply:
Great catch! To avoid any errors, I will switch to a tuple and then convert it to a numpy array to facilitate the `tf.Variable` assignment.

Reply:
Hi @LukeWood, I understand your point, but I'm finding it hard to see the code break even when I use an empty numpy array or a list. Can you elaborate a little on this or provide an example of where this fails?

For disclosure, the variables are erased at the end of the `text_to_image_prompt_to_prompt`.